examples/apm-sample/README.md

```diff
@@ -0,0 +1,23 @@
+# Raygun Crash Reporting + APM Sample
+
+This is a sample application to show how to use Raygun4Node and Raygun-APM together.
+
+This example uses the local `raygun4node` package in the project root directory by simply pointing to the root directory as a dependency in package.json:
```

Comment:
It would be good to add what feature of Raygun we further see light up and if other things are required for set up. E.g. do they have to follow the Raygun APM for Node instructions. I don't know if this provider is required to function the APM tool?

Reply:
There is some documentation in the root README.md as well. I think it would be better to reference that rather than duplicating the text.